test/built-ins/isFinite/prop-desc.js

```diff
@@ -8,7 +8,8 @@ description: >
 includes: [propertyHelper.js]
 ---*/
 
-verifyProperty(this, "isFinite", {
+verifyPrimordialCallableProperty(this, "isFinite", "isFinite", 1, {
+  value: this.isFinite,
```

Comment:
Let's not manually add these `value` properties

Reply:
I'm not 100% married to them, but this is not an original pattern and even appears in [INTERPRETING.md](https://github.com/tc39/test262/blob/main/INTERPRETING.md#includes).

Reply:
Removed.